Classes/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
No, this is not that easy. Currently, the script is expected to be in the root package only (at least when it is called), i.e. in the base distributions for the application installation, and in rest-api when testing the rest-api without the base-distribution. It currently relies on this fact to determine the root package path.

I'll use a different approach instead: I'm currently working on a function for determining the application root path. I'll then move the Composer ScriptHandler to the core package and use that from base-distribution, rest-api and web-frontend.